src/routes/+layout.svelte

```diff
@@ -44,6 +44,12 @@
 	scrollbarGutter="auto"
 >
 	<svelte:fragment slot="header">
+		<div class="flex justify-center">
```

Comment:
Can you explain this feature on Thursday. Not yet fully grasping it.

Reply:
This is just meant to make the skip-link appear centered on the page (and not cover the logo).